smoke/ip6_forward_test.sh

```diff
@@ -32,5 +32,5 @@ ip netns exec $p1 ping6 -i0.01 -c3 fd00:ba4:2::2
 ip netns exec $p2 ping6 -i0.01 -c3 fd00:ba4:1::2
 ip netns exec $p1 ping6 -i0.01 -c3 fd00:ba4:1::1
 ip netns exec $p2 ping6 -i0.01 -c3 fd00:ba4:2::1
-ip netns exec $p1 traceroute6 -N1 fd00:ba4:2::2
```

Comment:
Idem, it can go to a separate commit especially as you did not describe the reason for this change in the current commitlog.